go/enclave/rpc/rpc_encryption_manager.go

```diff
@@ -100,48 +95,3 @@ func (rpc *RPCEncryptionManager) EncryptWithViewingKey(address gethcommon.Addres
 
 	return encryptedBytes, nil
 }
-
-// ExtractTxHash - Returns the transaction hash from a common.EncryptedParamsGetTxReceipt object.
-func (rpc *RPCEncryptionManager) ExtractTxHash(encryptedParams common.EncryptedParamsGetTxReceipt) (gethcommon.Hash, error) {
```

Comment:
Stuff like this isn't strictly encryption related, and has been moved off into the utils class.